jobs/env/ci-kubernetes-e2e-scalability-common.env

```diff
@@ -8,7 +8,9 @@ LOGROTATE_MAX_SIZE=5G
 
 # Use IP-aliases for scalability tests.
 KUBE_GCE_ENABLE_IP_ALIASES=true
-# Turn on profiling for scheduler
+# Turn on profiling for various components.
+CONTROLLER_MANAGER_TEST_ARGS=--profiling
```

Comment:
Just to confirm - are you sure we're not already overriding this for was in high-speed env file?

Reply:
We do (similarly as we were for scheduler test args).
But let's solve that separately :)

Reply:
As long as it works, it's ok.